model/src/pyrenew/latent/infection_functions.py

```diff
@@ -112,30 +112,74 @@ def sample_infections_with_feedback(
         strength in time) or a vector representing
         the infection feedback strength at a
         given point in time.
-    generation_interval_pmf : ArrayLike
+    reversed_generation_interval_pmf : ArrayLike
         discrete probability mass vector
         representing the generation interval
-        of the infection process
-    infection_feedback_pmf : ArrayLike
+        of the infection process, where the final
+        entry represents an infection 1 time unit in the
+        past, the second-to-last entry represents
+        an infection two time units in the past, etc.
+    reversed_infection_feedback_pmf : ArrayLike
         discrete probability mass vector
-        whose `i`th entry represents the
-        relative contribution to infection
+        representing the infection feedback
+        process, where the final entry represents
+        the relative contribution to infection
         feedback from infections that occurred
-        `i` days in the past.
+        1 time unit in the past, the second-to-last
+        entry represents the contribution from infections
+        that occurred 2 time units in the past, etc.
 
     Returns
     -------
     tuple
-        A tuple `(Rt_adjusted, infections)`,
+        A tuple `(infections, Rt_adjusted)`,
         where `Rt_adjusted` is the infection-feedback-adjusted
         timeseries of the reproduction number R(t) and
         infections is the incident infection timeseries.
+
+    Notes
+    -----
+    This function implements the following renewal process:
+
+    .. math::
+
+        I(t) & = \mathcal{R}(t)\sum_{\tau=1}^{T_g}I(t - \tau)g(\tau)
+
+        \mathcal{R}(t) & = \mathcal{R}^u(t)\exp\left(\gamma(t)\
+            \sum_{\tau=1}^{T_f}I(t - \tau)f(\tau)\right)
+
+    where :math:`\mathcal{R}(t)` is the reproductive number,
+    :math:`\gamma(t)` is the infection feedback strength,
+    :math:`T_g` is the max-length of the
+    generation interval, :math:`\mathcal{R}^u(t)` is the raw reproduction
+    number, :math:`f(t)` is the infection feedback pmf, and :math:`T_f`
+    is the max-length of the infection feedback pmf.
+
+    Note that negative :math:`\gamma(t)` implies
+    that recent incident infections reduce :math:`\mathcal{R}(t)`
+    below its raw value in the absence of feedback, while
+    positive :math:`\gamma` implies that recent incident infections
+    _increase_ :math:`\mathcal{R}(t)` above its raw value, and
+    :math:`gamma(t)=0` implies no feedback.
+
+    In general, negative :math:`\gamma` is the more common modeling
+    choice, as it can be used to model susceptible depletion,
+    reductions in contact rate due to awareness of high incidence,
+    et cetera.
     """
```

Comment:
We should add back in 0-padding the reversed gen int and the reversed feedback pmf to be of the same length, and should add a check that the resultant pmfs are less than or equal to `I0` in length, matching the approach @damonbayer takes here: 

https://github.com/CDCgov/multisignal-epi-inference/blob/d0a773f5c5e5e608d0f1db5d43bfb48d64391739/model/src/pyrenew/latent/infections.py#L104-L116

Reply:
- `inf_feedback_strength` is padded in `latent.InfectionsWithFeedback.sample()` [here](https://github.com/CDCgov/multisignal-epi-inference/blob/db551da6dfce83e805d340f497f00db979a4252b/model/src/pyrenew/latent/infectionswithfeedback.py#L162-L167). 
- `i0` is padded in `model.RtInfectionsRenewalModel.sample()` [here](https://github.com/CDCgov/multisignal-epi-inference/blob/db551da6dfce83e805d340f497f00db979a4252b/model/src/pyrenew/model/rtinfectionsrenewalmodel.py#L288-L289).
- `inf_feedback_strength` can't be padded during `model.RtInfectionsRenewalModel.sample()` because it is sampled within `latent.InfectionsWithFeedback.sample()`.
- `i0` could be padded in either (model or component).
- Since the padding issue is, in principle, only relevant for `latent.InfectionsWithFeedback.sample()`, I would move the padding there.

Reply:
To clarify, this question is about `reversed_infection_feedback_pmf` and `reversed_generation_interval_pmf` as passed alongside `i0` to `sample_infections_with_feedback()`. They don't need to match in length _except_ for the double convolve, so I would favor deferring any needed padding of them until this point.